.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
```

Comment:
should this have `{'loose': true, 'modules': false}`

Reply:
No because it's only used to compile the `rollup.js` file. Babel in rollup is configured in `rollup.js`

Reply:
and by that I mean we don't use ES6 syntax at all right now in the main code.